front/lib/api/assistant/conversation.ts

```diff
@@ -243,7 +243,7 @@ export async function getUserConversations(
   }
 
   const participations = await ConversationParticipant.findAll({
-    attributes: ["userId", "createdAt"],
+    attributes: ["userId", "updatedAt"],
```

Comment:
Is that indexed?

Reply:
We still fetch via userId and action, there's no index on createdAt / updatedAt

Reply:
Shouldn't we add one?

Reply:
Ah yeah but we do order by updated at and maybe we want an index to help with that?